packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -99,26 +115,26 @@ export const GraphNG: React.FC<GraphNGProps> = ({
     const colorMode = getFieldColorModeForField(field);
     const seriesColor = colorMode.getCalculator(field, theme)(0, 0);
 
-    if (customConfig?.line?.show) {
+    if (customConfig?.mode !== GraphMode.Points) {
       seriesGeometry.push(
-        <Line
-          key={`line-${scale}-${i}`}
-          scaleKey={scale}
-          stroke={seriesColor}
-          width={customConfig?.line.show ? customConfig?.line.width || 1 : 0}
-        />
+        <Line key={`line-${scale}-${i}`} scaleKey={scale} stroke={seriesColor} width={customConfig?.lineWidth ?? 0} />
       );
     }
 
-    if (customConfig?.points?.show) {
+    if (customConfig?.points !== PointMode.Never) {
```

Comment:
the Points.Auto does not work, always shows points. Think we need to pass this auto mode to Points and define show function (uPlot calls this) to make this dynamic. Or we can check data point count here and make this part dynamic

Reply:
Getting `PointMode.Auto` to actually work is outside the scope of this PR -- that will take some digging into the react bindings.  these are settings on the uplot config object, but I'm not sure how we expose them via react yet

Reply:
auto | always should be passed to the configs below